### ISSUE TITLE
Apply elemento typed events

### ISSUE THREAD

**Issue by ibaca** [TDesjardins/gwt-ol3#106]
This is more an experiment. OL does not use standard Event, so its events do not extend DOM Event (not totally sure this sentence makes sense). Anyway, this issue is to investigate if it is possible and useful to use the Element EventType interface, and so elemental2 EventTarget and EventListener to describe the event. The best of this approach is event unification and rxgwt integration 😉.

**Comment by TDesjardins** [comment]
I currently created some feature branches. One of them is for testing integration of elemental2 (see https://github.com/TDesjardins/gwt-ol3/tree/elemental2_integration). This is especially to use future proofed XML- and HTML-DOM stuff. This could be the starting point for additional modifications on elemental things.

**Comment by ibaca** [comment]
Yep, hehe I saw it this weekend, and it was what get me motivated to start investigating about elemento 👍.
